src/pages/workspaces/workspace/workflows/LaunchAnalysisModal.js

```diff
@@ -49,6 +50,8 @@ const LaunchAnalysisModal = ({
   const [bucketLocation, setBucketLocation] = useState({});
   const [userComment, setUserComment] = useState(undefined);
   const [userCommentError, setUserCommentError] = useState(undefined);
+  // Currently, default backend is LifeSciences if there is no 'UseCromwellGcpBatchBackend' setting found for a workspace
```

Comment:
Do you want to add a TODO with the corresponding JIRA ticket to switch that logic at the end of milestone 3 (Batch is default)?